bin/junit/all-tests.html

```diff
@@ -0,0 +1,25 @@
+<html xmlns:string="xalan://java.lang.String" xmlns:lxslt="http://xml.apache.org/xslt">
```

Comment:
Please make use of .gitignore file. There are a lot of unnecessary file commits.